public/app/plugins/datasource/mssql/partials/config.html

```diff
@@ -42,10 +42,16 @@ <h3 class="page-heading">MS SQL connection</h3>
 		</div>
 	</div>
 
+</div>
+
+<h3 class="page-heading">TLS/SSL Auth</h3>
+
+<div class="gf-form-group">
+
 	<div class="gf-form">
-		<label class="gf-form-label width-7" for="encrypt-select">Encrypt</label>
```

Comment:
Unintentional removal of aria attributes? Please fix

Reply:
yes, was not intended. fixed now